.gitignore

```diff
@@ -0,0 +1,278 @@
+# Top level
+__pycache__/*
+
+# Global
+*/*/.vscode/*
+
+
+
+
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# # # # # # # # # # # # #     Backend Python Services     # # # # # # # # # # # # #
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
```

Comment:
We can avoid having to preprend ever line with `backend` by simply creating a `.gitignore` in the `backend` directory. Similarly with `frontend`. We should only have things in this gitignore that are common amongt all subdirectories

Reply:
Ooh yeah, great shout! Did't cross my mind!